### PR TITLE
Feature chinedu request a service

### PR DESCRIPTION
**Pull request by ChineduOguejiofor**
#### What does this PR achieve?
- User can request a service
- Adds the ability to detect user location
- User can make a request based on saved locations
- Order summary has been properly formatted
- 

#### Screenshots:
order

#### Relevant Trello Tickets
- [User Request Service](https://trello.com/c/eMFyfPHI/6-user-dashboard-request-a-service-frontend)

### CODE REVIEW

File: src/components/userDashboard/components/profile/actionTypes.js [hunk 1]
```diff
@@ -8,9 +8,13 @@ const actionTypes = {
     GET_RECORD: 'GET_RECORD',
     GET_RECORD_START: 'GET_RECORD_START',
     SAVE_RECORD: 'SAVE_RECORD',
+    SET_ADDRESS: 'SET_ADDRESS',
     SET_GEO_LOCATION: 'SET_GEO_LOCATION',
     UPDATE_ADDRESS: 'UPDATE_ADDRESS',
     UPDATE_RECORD: 'UPDATE_RECORD',
+    UPLOAD_ADDRESS_FAILED: 'UPLOAD_ADDRESS_SUCCESS',
```
Comment (justiceotuya) [inline src/components/userDashboard/components/profile/actionTypes.js]: ```UPLOAD_ADDRESS_FAILED: 'UPLOAD_ADDRESS_SUCCESS',```
check this. it is wrong

File: src/components/userDashboard/components/profile/actions.js [hunk 2]
```diff
@@ -3,10 +3,10 @@ import actionTypes from './actionTypes';
 const {
     ADD_ADDRESS,
```
Comment (justiceotuya) [inline src/components/userDashboard/components/profile/actions.js]: They should all be in a seperate line

File: src/components/userDashboard/components/profile/actionTypes.js (hunk 1 above)
Comment (ChineduOguejiofor) [reply src/components/userDashboard/components/profile/actionTypes.js]: My bad
Updated
Thanks

File: src/components/userDashboard/components/profile/actions.js (hunk 2 above)
Comment (ChineduOguejiofor) [reply src/components/userDashboard/components/profile/actions.js]: updated
Thanks

File: src/components/userDashboard/components/profile/sagas.js [hunk 3]
```diff
@@ -26,12 +32,28 @@ function* updateRecord({ payload }) {
...
+            payload: user,
+            type: PROFILE_UPDATE,
+        });
+    } catch (error) {
+        const response = yield error.response;
+        yield put({
+            payload: response.data.error,
+            type: UPLOAD_ADDRESS_FAILED,
+        });
+    }
```
Comment (olajohn-ajiboye) [inline src/components/userDashboard/components/profile/sagas.js]: Nice destructuring

**Pull request merged**